package.json

```diff
@@ -6,20 +6,26 @@
   "private": true,
   "scripts": {
     "dev": "nuxt",
-    "build": "nuxt build",
+    "build": "nuxt build --target static && nuxt export",
```

Comment:
What is this doing?

Reply:
I think it does the same as `nuxt generate`. Was more for me to test. Also see my [previous comment](https://github.com/lucko/LuckPermsWeb/pull/283#issuecomment-651070482).